<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="contextMenuPolicy">
    <enum>Qt::ContextMenuPolicy::NoContextMenu</enum>
   </property>
   <widget class="QPushButton" name="buttonUndo">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>280</y>
      <width>181</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 19, 102);</string>
    </property>
    <property name="text">
     <string>悔棋</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonReset">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>390</y>
      <width>181</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>重新开始</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonAIFirst">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>20</y>
      <width>171</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>电脑先手</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonUserFirst">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>130</y>
      <width>171</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>玩家先手</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>786</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
